web/react/stores/preference_store.jsx

```diff
@@ -122,5 +122,6 @@ class PreferenceStoreClass extends EventEmitter {
 }
 
 const PreferenceStore = new PreferenceStoreClass();
+PreferenceStore.setMaxListeners(0);
```

Comment:
That's fine, it's a very common practice in React and really isn't that slow.

https://facebook.github.io/react/docs/multiple-components.html#a-note-on-performance